src/StateService/Settings.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Linq;
+
+namespace Neo.Plugins.StateService
+{
+    internal class Settings
+    {
+        public string Path { get; }
+        public bool FullState { get; }
+        public bool StartValidate { get; }
```

Comment:
Where is it used?

Reply:
In state validate. Not in this pr. I can remove these.

Reply:
Cleaned.